web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -106,10 +106,8 @@ describe("<LicenseTask />", () => {
 
     it("shows content on first tab when lastUpdatedISO is empty", () => {
       useMockBusiness({
-        licenseData: generateLicenseData(
-          {},
-          { [taskIdLicenseNameMapping[task.id]]: generateLicenseDetails({ lastUpdatedISO: "" }) }
-        ),
+        licenseData: undefined,
```

Comment:
pebble - we should keep license data, I'd say this, because there is logic tied to lastUpdatedISO within LicenseData, we provide lastUpdatedISO within each license result.

Reply:
I just realized your using lastUpdatedISO that is defined withing the business object